roles/common/templates/monitoring/config.json

```diff
@@ -6,7 +6,11 @@
       "openstack"
     ],
     "keepalive": {
-      "handler": "{{ monitoring.keepalive.handler }}"
+      "handler": "{{ monitoring.keepalive.handler }}",
+	  "thresholds": {
```

Comment:
if we're adding thresholds then we should make them configurable.  something like:

```
monitoring:
  keepalive:
    thresholds:
      warning: 40
      critical: 60
```